cli/src/command/account.rs

```diff
@@ -920,81 +957,122 @@ pub async fn voting_output_command(account: &Account) -> Result<(), Error> {
-                    if let Some(nts) = output_data.output.native_tokens() {
-                        native_tokens.add_native_tokens(nts.clone())?;
-                    }
-                    match &output_data.output {
-                        Output::Nft(nft) => nfts.push(nft.nft_id_non_null(output_id)),
-                        Output::Alias(alias) => aliases.push(alias.alias_id_non_null(output_id)),
-                        Output::Foundry(foundry) => foundries.push(foundry.id()),
-                        Output::Basic(_) | Output::Treasury(_) => {}
-                    }
-                    let unlock_conditions = output_data
-                        .output
-                        .unlock_conditions()
-                        .expect("output must have unlock conditions");
-                    let sdr_amount = unlock_conditions
-                        .storage_deposit_return()
-                        .map(|sdr| sdr.amount())
-                        .unwrap_or(0);
-
-                    amount += output_data.output.amount() - sdr_amount;
-                }
+    for (output_id, output_data) in unspent_outputs.into_iter().map(|data| (data.output_id, data)) {
+        outputs.push((output_id, output_data.output.kind_str().to_string()));
+
+        // Output might be associated with the address, but can't be unlocked by it, so we check that here.
+        // Panic: cannot fail for outputs belonging to an account.
+        let (required_address, _) = output_data
+            .output
+            .required_and_unlocked_address(current_time, &output_id, None)
+            .unwrap();
+
+        if address.inner() == &required_address {
+            if let Some(nts) = output_data.output.native_tokens() {
+                native_tokens.add_native_tokens(nts.clone())?;
             }
+            match &output_data.output {
+                Output::Alias(alias) => aliases.push(alias.alias_id_non_null(&output_id)),
+                Output::Nft(nft) => nfts.push(nft.nft_id_non_null(&output_id)),
+                Output::Basic(_) | Output::Foundry(_) | Output::Treasury(_) => {}
+            }
+            let unlock_conditions = output_data
+                .output
+                .unlock_conditions()
+                .expect("output must have unlock conditions");
+            let sdr_amount = unlock_conditions
+                .storage_deposit_return()
+                .map(|sdr| sdr.amount())
+                .unwrap_or(0);
+
+            amount += output_data.output.amount() - sdr_amount;
         }
     }
 
-    log = format!(
-        "{log}\nOutputs: {:#?}\nBase coin amount: {}\nNative Tokens: {:#?}\nNFTs: {:#?}\nAliases: {:#?}\nFoundries: {:#?}\n",
-        output_ids,
-        amount,
-        native_tokens.finish_vec()?,
-        nfts,
-        aliases,
-        foundries,
-    );
+    // base tokens table
+    println_log_info!("Base Tokens");
+    println_log_info!("{:<8}{}", "Amount:", amount);
```

Comment:
I'd put these two on the same line

Reply:
yeah, fine with that now ... the table isn't format-consistent anymore anyway.